other/exposed-prometheus-metrics.bcheck

```diff
@@ -0,0 +1,23 @@
+metadata:
+    language: v1-beta
+    name: "Prometheus Metrics Publicly Available"
+    description: "Prometheus Metrics Found"
+    author: "Sourav Kalal"
+
+run for each:
```

Comment:
@Anof-cyber we could just use a "define" block here instead of a run for each since we are only testing the one path. 

We'd welcome and accept any future PRs cleaning this up 😄